docs/building-modules/keeper.md

```diff
@@ -72,7 +72,7 @@ and go through the following steps:
 2. Marhsall `value` to `[]byte` using the codec `cdc`. 
 3. Set the encoded value in the store at location `key` using the `Set(key []byte, value []byte)` method of the store. 
 
-For more, see an example of `keeper`'s [methods implementation from the nameservice tutorial](https://github.com/cosmos/sdk-application-tutorial/blob/c6754a1e313eb1ed973c5c91dcc606f2fd288811/x/nameservice/internal/keeper/keeper.go). 
+For more, see an example of `keeper`'s [methods implementation from the `staking` module](https://github.com/cosmos/cosmos-sdk/blob/3bafd8255a502e5a9cee07391cf8261538245dfd/x/staking/keeper/keeper.go). 
 
 ## Next {hide}
```

Comment:
It'd be nice to explain which store to use (the preferred one being a prefix store). Also, the iterators are missing from this file, as it only explains how to retrieve single value objects from the store.

Reply:
added